prefs.js

```diff
@@ -112,9 +112,29 @@ function buildPrefsWidget() {
   );
   prefsWidget.attach(feedbackSoundsSwitch, 1, 4, 1, 1);
 
+  const playingSoundNotSupportedLabel = new Gtk.Label({
+    halign: Gtk.Align.START,
+    visible: !is_playing_sound_supported(),
```

Comment:
This doesn't have any affect on the visibility, at least not on my machine. Instead you could add the label if playing is not supported. I think if you add the switch if it is supported and the warning message if not, it's even clearer to the user that sound can not be enabled. You could also put the label in the same position as the switch, but having it below also works.

Reply:
Hmm, strange, It believe I checked it.

Ok, it seems that using `set_markup` "breaks" `visible`. Without that it works as expected. I switched to `set_visible()` and it works also with `set_markup()`.

Your proposal it definitely more user friendly, however, it is more complicated (with my zero-to-none skills in JavaScript :-) ). Eventually, it applied `set_sensitive()`, however, the state can be still "on":
![image](https://user-images.githubusercontent.com/148013/160296543-e81c40ee-9e26-4c00-ae3b-bba48a19efdd.png)

Once the dependency is installed, it should just play the sounds (without switching it on manually).